atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -275,19 +276,22 @@ def run(self):
         """
         run the plugin
         """
-        try:
-            config = self.workflow.base_image_inspect[INSPECT_CONFIG]
-        except (AttributeError, TypeError):
-            message = "base image was not inspected"
-            self.log.error(message)
-            raise RuntimeError(message)
-        else:
-            base_image_labels = config["Labels"] or {}
-
         dockerfile = df_parser(self.workflow.builder.df_path, workflow=self.workflow)
 
         lines = dockerfile.lines
 
+        if re.match('^koji/image-build(:.*)?$', dockerfile.baseimage):
```

Comment:
I'd rather this case wasn't called out like this...
Why can't we make inspection of base image optional?
Falling back to an empty dict on `KeyError` seems like a much better solution.